crates/chia-secp/src/secp256k1/public_key.rs

```diff
@@ -0,0 +1,59 @@
+use std::fmt;
+use std::hash::{Hash, Hasher};
+
+use chia_sha2::Sha256;
+use k256::ecdsa::signature::hazmat::PrehashVerifier;
+use k256::ecdsa::{Error, VerifyingKey};
+
+use super::Secp256k1Signature;
+
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub struct Secp256k1PublicKey(pub(crate) VerifyingKey);
```

Comment:
you don't think this is a bit long name for the type? I would think the secp256-part is implied by the crate it's defined in. maybe k1 is awkward on its own though

Reply:
K1PublicKey isn't bad I suppose

Reply:
Changed this